theme/docs.less

```diff
@@ -472,3 +472,14 @@
     }
 }
 .replaceDocsAvatar();
+
+#docs.py {
```

Comment:
does this apply to javascript too?

Reply:
yeah you're right, i think it should. updated to use a single editorlang-text class to style both js/python